modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/cluster/ClusterRequestExecutorFixture.java

```diff
@@ -30,21 +31,22 @@ public void setUp() {
 		ClusterHealthStatusTranslator clusterHealthStatusTranslator =
 			new ClusterHealthStatusTranslatorImpl();
 
-		_clusterRequestExecutor = new ElasticsearchClusterRequestExecutor() {
-			{
-				setHealthClusterRequestExecutor(
-					_createHealthClusterRequestExecutor(
-						clusterHealthStatusTranslator,
-						_elasticsearchClientResolver));
-				setStateClusterRequestExecutor(
-					_createStateClusterRequestExecutor(
-						_elasticsearchClientResolver));
-				setStatsClusterRequestExecutor(
-					_createStatsClusterRequestExecutor(
-						clusterHealthStatusTranslator,
-						_elasticsearchClientResolver));
-			}
-		};
+		ClusterRequestExecutor clusterRequestExecutor =
+			new ElasticsearchClusterRequestExecutor();
+
+		ReflectionTestUtil.setFieldValue(
+			clusterRequestExecutor, "_healthClusterRequestExecutor",
+			_createHealthClusterRequestExecutor(
+				clusterHealthStatusTranslator, _elasticsearchClientResolver));
+		ReflectionTestUtil.setFieldValue(
+			clusterRequestExecutor, "_stateClusterRequestExecutor",
+			_createStateClusterRequestExecutor(_elasticsearchClientResolver));
+		ReflectionTestUtil.setFieldValue(
+			clusterRequestExecutor, "_statsClusterRequestExecutor",
+			_createStatsClusterRequestExecutor(
+				clusterHealthStatusTranslator, _elasticsearchClientResolver));
+
+		_clusterRequestExecutor = clusterRequestExecutor;
```

Comment:
@W2jx I'm still seeing such usages in several places in this PR. What I meant is, why not

```
_clusterRequestExecutor = new ElasticsearchClusterRequestExecutor();

ReflectionTestUtil.setFieldValue(_clusterRequestExecutor, xxx, xxx);
```

Reply:
I get it, and I will fix them.